application/src/main/java/org/togetherjava/tjbot/config/FreeCommandConfig.java

```diff
@@ -0,0 +1,60 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Config instance for the Free Command System see
+ * {@link org.togetherjava.tjbot.commands.free.FreeCommand}
+ *
+ * The Json looks as follows:
+ * 
+ * <pre>
+ * "freeCommand": [
+ *   {
+ *       "statusChannel": long_number,
+ *       "monitoredChannels": [long_number, long_number]
+ *   }]
+ * </pre>
+ * 
+ * Additional Guilds may add their settings by adding additional {"statusChannel....... }
+ *
+ * The long channel ID can be found by right-clicking on the channel and selecting 'Copy ID'
+ */
+@SuppressWarnings("ClassCanBeRecord")
+@JsonRootName(value = "freeCommand")
```

Comment:
(root name change is obsolete if changing what i said earlier)

Reply:
I think ur confusing me and notnull?
ur previous instructions include adding (JsonCreator.mode = PROPERTIES) and changing to List and making unmodifiable.

Nothing about changing class name or structure or anything else i can think of that will remove that anotation

Reply:
yeah, sorry for that. Anyways, the `value =` is obsolete, i think. Just `@JsonRootName("freeCommand")` is enough (afaik its the default key).

Reply:
yeah, it works without `value=`

fixed